gatsby-theme-marketing/src/components/TroubleshootSpec.js

```diff
@@ -286,7 +286,7 @@ class TroubleshootSpec extends React.Component {
               </div>
 
               {!isMobile ?
-                <div className="AbsoulteLintExpressions-wrapper flex-column u-width--third u-marginRight--10">
+                <div className="AbsoulteLintExpressions-wrapper flex-column u-width--fourth u-marginRight--10">
```

Comment:
also maybe we don't need separate util for fourth if it's just used here and not anywhere else.
can you just add style={{width: 25%}} to this div?